src/Sentry/Platforms/Native/UnmanagedHttpContent.cs

```diff
@@ -0,0 +1,63 @@
+using Sentry.Extensibility;
+
+namespace Sentry.Native;
+
+internal class UnmanagedHttpContent : SerializableHttpContent
+{
+    private readonly IntPtr _content;
+    private readonly int _length = 0;
+    private readonly IDiagnosticLogger? _logger;
+
+    public UnmanagedHttpContent(IntPtr content, int length, IDiagnosticLogger? logger)
+    {
+        _content = content;
+        _length = length;
+        _logger = logger;
+    }
+
+    protected override Task SerializeToStreamAsync(Stream stream, TransportContext? context)
+    {
+        try
+        {
+            unsafe
+            {
+                using var unmanagedStream = new UnmanagedMemoryStream((byte*)_content.ToPointer(), _length);
```

Comment:
question: The address of `_content` points to unmanaged memory, right? So not concerns about pinning?